README.md

```diff
@@ -50,45 +56,47 @@ export class UserController {
 }
 ```
 
-To create your first LoopBack 4 application, See [Getting Started](http://loopback.io/doc/en/lb4/Getting-started.html).
-
+To create your first LoopBack 4 application, See
+[Getting Started](http://loopback.io/doc/en/lb4/Getting-started.html).
 
 # Documentation
 
-- [Official documentation](http://loopback.io/doc/en/lb4/)
-- [API documentation](http://apidocs.loopback.io/#LoopBack4)
-- [FAQ](http://loopback.io/doc/en/lb4/FAQ.html)
-- [LoopBack 3 vs LoopBack 4](http://loopback.io/doc/en/lb4/LoopBack-3.x.html)
-- [Tutorials and examples](http://loopback.io/doc/en/lb4/Examples-and-tutorials.html)
+* [Official documentation](http://loopback.io/doc/en/lb4/)
+* [API documentation](http://apidocs.loopback.io/#LoopBack4)
+* [FAQ](http://loopback.io/doc/en/lb4/FAQ.html)
+* [LoopBack 3 vs LoopBack 4](http://loopback.io/doc/en/lb4/LoopBack-3.x.html)
+* [Tutorials and examples](http://loopback.io/doc/en/lb4/Examples-and-tutorials.html)
 
 # Contributing
 
 See the following resources to get your started:
 
- - [Contributing Guidelines](./docs/CONTRIBUTING.md)
- - [Monorepo overview](./docs/site/MONOREPO.md)
- - [Developing LoopBack](./docs/site/DEVELOPING.md)
+* [Contributing Guidelines](./docs/CONTRIBUTING.md)
+* [Monorepo overview](./docs/site/MONOREPO.md)
+* [Developing LoopBack](./docs/site/DEVELOPING.md)
 
-You can join the team by posting a comment to [issue #110](https://github.com/strongloop/loopback-next/issues/110.).
+You can join the team by posting a comment to
+[issue #110](https://github.com/strongloop/loopback-next/issues/110.).
 
 # Team
 
 ## Project Architects:
 
-Raymond Feng|Miroslav Bajtos|Ritchie Martori|Kevin Delisle
-:-:|:-:|:-:|:-:
-[<img src="https://avatars0.githubusercontent.com/u/540892?v=3&s=60">](http://github.com/raymondfeng)|[<img src="https://avatars2.githubusercontent.com/u/1140553?v=3&s=60">](http://github.com/bajtos)|[<img src="https://avatars2.githubusercontent.com/u/462228?v=3&s=60">](http://github.com/ritch)|[<img src="https://avatars3.githubusercontent.com/u/2053534?v=3&s=60">](http://github.com/kjdelisle)
+|                                             Raymond Feng                                              |                                          Miroslav Bajtos                                          |                                         Ritchie Martori                                         |                                            Kevin Delisle                                             |
```

Comment:
The readability is pretty horrendous here. Should we configure prettier so that it doesn't run on tables?

Reply:
I'm not sure how to disable it for just tables in `md` files. Any ideas?

Reply:
I don't think it's possible to do it just for tables in prettier. Curious to hear others' opinion on this.

Reply:
Perhaps https://github.com/prettier/prettier/issues/3211 might help.

Reply:
Great fine! I've updated the tables (with the exception of @shimks 's image since I'm not sure how to set the height attribute in reference style and GitHub's `s=60` isn't working for his Avatar).